gnomad_qc/v4/resources/variant_qc.py

```diff
@@ -1,6 +1,6 @@
 """Script containing variant QC related resources."""
 
-from typing import Union
+from typing import Optional, Union
```

Comment:
Optional isn't needed in this case, it's only needed when you want to support `None`
```suggestion
from typing import Union
```